LumpNav/LumpNav.py

```diff
@@ -228,7 +285,7 @@ def setupScene(self): #applet specific
     if not self.cauteryTipToCautery:
       self.cauteryTipToCautery=slicer.vtkMRMLLinearTransformNode()
       self.cauteryTipToCautery.SetName("CauteryTipToCautery")
-      m = self.readTransformFromSettings('CauteryTipToCautery')
+      m = self.readTransformFromSettings('Configurations/' + self.parameterNode.GetParameter('Configuration') + '/CauteryTipToCautery') 

```

Comment:
This is not nice. ReadTransformFromSettings should hide all the details. Also ReadTransformFromSettings should first try to read value from the currently selected configuration and if it does not find the value there then it should try to get it from default. This allows adding new values without the need of patching existing setting files.